File: KITTY/KURS_1/JAVA/DAY_18_Two_Browser/app/src/main/java/com/example/velic/day_18_two_browser/SecondActivity.java
package com.example.velic.day_18_two_browser;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.view.View;
import android.webkit.WebView;
import android.webkit.WebViewClient;

public class SecondActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        Uri url = getIntent().getData();
        WebView webView = (WebView) findViewById(R.id.webView);
        webView.setWebViewClient(new Callback());
        webView.loadUrl(url.toString());
    }
    private class Callback extends WebViewClient {
        @Override
        public boolean shouldOverrideUrlLoading
                (WebView view, String url) {
            return(false);
        }
    }
}

File: KITTY/KURS_1/KOTLIN/DAY_18_2_Browser_default_kt/app/src/main/java/com/example/day_18_2_browser_default_k/SecondActivity.kt
package com.example.day_18_2_browser_default_k

import android.os.Bundle
import android.webkit.WebResourceRequest
import android.webkit.WebView
import android.webkit.WebViewClient

import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_second.*

class SecondActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)

        val url = intent.data
        webView.webViewClient = Callback()
        webView.loadUrl(url!!.toString())
    }

    private inner class Callback : WebViewClient() {
        override fun shouldOverrideUrlLoading(view : WebView ,request : WebResourceRequest): Boolean {
            return false
        }
    }
}
